node/tests/integration/main.rs

```diff
@@ -486,6 +573,42 @@ async fn assert_slot_status(
     );
 }
 
+async fn assert_slot_status_call(
+    client: &AzeClient,
+    account_id: AccountId,
+    slot_data: GameStorageSlotData
+) {
+    let (account, _) = client.get_account(account_id).unwrap();
+    let game_account_storage = account.storage();
+
+    let small_blind_amt = slot_data.small_blind_amt();
+    let highest_bet = slot_data.highest_bet();
+
+    let mut slot_index = 54;
+
+    // checking the small blind amount
+    assert_eq!(
+        game_account_storage.get_item(slot_index),
+        RpoDigest::new([Felt::from(small_blind_amt), Felt::ZERO, Felt::ZERO, Felt::ZERO])

```

Comment:
Why are we asserting on small blind amount and highest bet, Any reasons here. Since in the procedure we are just changing the remaining balance slot of the player.

Reply:
If we don't need this assert let's remove this, rest of the changes LGTM